MAIN_minimalistic.m

```diff
@@ -0,0 +1,40 @@
+function MAIN_minimalistic()
+close all; clear; clc;
+
+%% Physical parameters
+g = 9.81; % (m/s^2) gravitational acceleration
+m = 0.3;  % (kg) mass of the ball
+c = 0.02; % (N*s^2/m^2) quadratic drag coefficient
+
+e = 0.95; % Coefficient of restitution
+P.rollingThreshold = 1e3;
+
+y0 = [0-1i 2-10i]'; % Initial state (x,y position. x,y velocity)
+
+options = odeset('RelTol',1e-6,'AbsTol',1e-6,...
+    'Events',@EventFunction_minimalistic,'MaxStep',0.1);
+
+% Acceleration due to gravity and quadratic decelatation due to drag
+dynamics_function = @(t,y) [y(2); -c*y(2).*norm(y(2))/m - g*1i];
```

Comment:
It would be great to break this out into a function call:  `dynamics_function = @(t, y)(dynFun_minimalistic(T));` and then add a file: `dynFun_minimalistic.m`. Feel free to pick any reasonable name.